### PR TITLE
Chore/fix wysiwyg preview

### LINKED ISSUE #6051: Nested lists in Rich text field (markdown) doesn't work
**Describe the bug**

Nested lists in Rich text field (markdown) doesn't work

**Steps to reproduce the behavior**

1. Create a rich text field
2. Create a nested list
3. The markdown preview show just one list, without nested

Screenshot 2020-05-04 at 18 26 37

**Expected behavior**

Show the nested lists:

Screenshot 2020-05-05 at 09 53 22

**System**

- Node.js version: v12
- NPM version: v6
- Strapi version: v3.0.0-beta.20.1
- Database: MongoDB
- Operating system: macOS

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:
This PR removes draft-js in favor of ReactMarkdown in order to display the entered Markdown in the content manager plugin.

Fixes #6051 .

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
@soupette can we get a screenshot of the new editor look?

**Comment by soupette** [comment]
> @soupette can we get a screenshot of the new editor look?

@derrickmehaffy it is the exact same one except that the preview now supports tables, nested lists, nested quotes and also code block syntax highlighting

File: packages/strapi-plugin-content-manager/admin/src/components/InputJSON/index.js [hunk 1]
```diff
@@ -13,15 +13,16 @@ import 'codemirror/addon/lint/javascript-lint';
 import 'codemirror/addon/edit/closebrackets';
 import 'codemirror/addon/selection/mark-selection';
 import 'codemirror/lib/codemirror.css';
-import 'codemirror/theme/3024-night.css';
+// import 'codemirror/theme/3024-night.css';
```
Comment (petersg83) [inline packages/strapi-plugin-content-manager/admin/src/components/InputJSON/index.js:16]: can we remove this line?

File: packages/strapi-plugin-content-manager/package.json [hunk 2]
```diff
@@ -13,21 +13,22 @@
...
+    "highlight.js": "^10.1.1",
     "immutable": "^3.8.2",
     "lodash": "^4.17.11",
     "pluralize": "^7.0.0",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "react-intl": "4.5.0",
     "react-fast-compare": "^3.2.0",
+    "react-intl": "4.5.0",
+    "react-markdown": "^4.3.1",
```
Comment (petersg83) [inline packages/strapi-plugin-content-manager/package.json:24]: Should we remove `^`? Same for all packages.

@alexandrebodin what do you think?

**Pull request merged**